README.md

```diff
@@ -3,41 +3,136 @@ Module of the Viam vision service to automatically build, deploy, and run infere
 
 Internally, this module calls DirectAI's `deploy_detector` and `deploy_classifier` endpoints with the provided configuration, saves the ID of the deployed models, and then calls the models via their IDs on incoming images. See [docs](https://api.alpha.directai.io/docs) for an overview of DirectAI's publicly available API.
 
-## Getting started
+Note that this README closely follows Viam's in-house motion-detector documentation, as described [here](https://github.com/viam-labs/motion-detector/).
 
-The first step is to obtain DirectAI API credentials. Instructions [here](https://api.alpha.directai.io/docs).
-
-This module implements the following methods of the [vision service API](https://docs.viam.com/services/vision/#api):
-  * `GetDetections()`
-  * `GetDetectionsFromCamera()`
-  * `GetClassifications()`
-  * `GetClassificationsFromCamera()`
+## Getting started with Viam & DirectAI
 
+Start by [configuring a camera](https://docs.viam.com/components/camera/webcam/) on your robot. Remember the name you give to the camera, it will be important later.
 > [!NOTE]  
 > Before configuring your vision service, you must [create a robot](https://docs.viam.com/manage/fleet/robots/#add-a-new-robot).
 
+Before calling DirectAI's API via Viam, you have to grab your client credentials. Generate them by clicking **Get API Access** on the [DirectAI website](https://directai.io/). Save them in an Access JSON on the same machine that's running your Viam Server.
+```json
+{
+  "DIRECTAI_CLIENT_ID": "UE9S0AG9KS4F3",
+  "DIRECTAI_CLIENT_SECRET": "L23LKkl0d5<M0R3S4F3"
+}
+```
+
 ## Configuration
 
 Navigate to the **Config** tab of your robot’s page in [the Viam app](https://app.viam.com/). Click on the **Services** subtab and click **Create service**. Select the `vision` type, then select the `directai-beta` model. Enter a name for your service and click **Create**.
 
-### Example Configuration
+On the new component panel, copy and paste the Example Detector *or* Classifier Attributes. Ensure that the Access JSON path that you provide in your Config is **absolute**, not relative. (e.g. `/Users/janesmith/Downloads/directai_credential.json`, not `~/Downloads/directai_credential.json`)
```

Comment:
should be able to do both detector and classifier in one json

Reply:
I opted to show one at a time for simplicity but I can make a mention of it in the description. happy to discuss more if need be

Reply:
update: made a note